test/integration/testsuite_test.go

```diff
@@ -31,6 +31,10 @@ func TestTest(t *testing.T) {
 
 	// fetch the current (reporter) config
 	_, reporterConfig := GinkgoConfiguration()
+	err := os.Mkdir("out", 0775)
```

Comment:
I should use `os.MkdirAll` because in case `out` directory already exist then this will through an error but `MkdirAll` will not.
```
// If path is already a directory, MkdirAll does nothing
// and returns nil.
```